src/Majora/Framework/Domain/Action/Dal/DalActionTrait.php

```diff
@@ -95,4 +95,28 @@ protected function fireEvent($eventName, Event $event)
 
         $this->eventDispatcher->dispatch($eventName, $event);
     }
+
+    /**
+     * @see NormalizableInterface::getScopes()
+     */
+    public static function getScopes()
+    {
+        return array();
+    }
+
+    /**
+     * @see NormalizableInterface::normalize()
+     */
+    public function normalize($scope = 'default')
+    {
+        return array();
```

Comment:
I don't understand what is the purpose of this Trait? 
it seems to be even more useless than the `StdNormalizable` class